x-pack/plugins/threat_intelligence/scripts/generate_indicators.js

```diff
@@ -50,7 +50,7 @@ const main = async () => {
             'threat.feed.name': {
               type: 'keyword',
             },
-            'threat.indicator.url.original': {
+            'threat.indicator.url.full': {
```

Comment:
good catch, I guess I forget to modify these ones in my investigate in timeline PR, sorry!